src/_test_/App.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import Country from '../Pages/Country';
+import store from '../Redux/store';
+
+it('Countries renders correctly', () => {
+  const countries = renderer
+    .create(<Provider store={store}><Country /></Provider>)
+    .toJSON();
+  expect(countries).toMatchSnapshot();
+});
```

Comment:
- [ ] Great job working on the project I really appreciate your hard work but here I can see you just made tests for app.js files but it is required to test for pure functions in your Redux code. so kindly add more tests to test your app since the tests are very important to tests that everything is working properly 🔥🔥


![image](https://user-images.githubusercontent.com/58553711/206883238-f68d7bd6-c259-43d6-a554-4685b430960e.png)